src/test/java/hudson/plugins/git/TestGitRepo.java

```diff
@@ -34,9 +35,9 @@ public class TestGitRepo {
     public TestGitRepo(String name, File tmpDir, TaskListener listener) throws IOException, InterruptedException {
 		this.name = name;
 		this.listener = listener;
-		
-		EnvVars envVars = new EnvVars();
-		
+
+        EnvVars envVars = GitUtilsTest.getConfigNoSystemEnvsVars();
```

Comment:
well the code here use tabs whereas I'm using spaces :)

Reply:
I adjusted it to reduce white space diffs as mentioned in the contributing guide.  Thanks very much for the fixes!

Reply:
thanks.
fair enough. but it's not really convenient to have files using tabs and files using spaces within the same project.
Intellij doesn't have a per file configuration feature :)